app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { CONTACTS } from "./constants";
+import DeleteAlert from "./DeleteAlert";
+import Menubar from "./Menubar";
+import Table from "./Table";
+
+const Contacts = () => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [showMenubar, setShowMenuBar] = useState(true);
+  const [contacts, setContacts] = useState(CONTACTS);
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [selectedContactId, setSelectedContactId] = useState();
```

Comment:
Big Anti-pattern. No state should be declared without initial values.

Reply:
@yedhink  Fixed with #14